### ISSUE TITLE
bug "... is not a win32 application" when start up

### ISSUE THREAD

**Issue by alanganteng** [i486/VxKex#6]
it appears every time i starting up the windows, the settings has check the bypass "... is not a win32 application" error message. but it didn't work. please fix that

**Comment by CommonLoon102** [comment]
Try this. I haven't tried myself, I don't know if it is safe, I've just heard that this can solve the issue.

https://board.eclipse.cx/viewtopic.php?t=652

**Comment by casemap77** [comment]
Hi @CommonLoon102  thanks for the info and winvertroll does solve the issue of running latest VS Code v1.92.0 by getting rid of the startup error. :)

**Comment by ask2018** [comment]
I have the same issue with latest version of ElectronMail 5.2.3 based on Electron 32.0.1

https://github.com/vladimiry/ElectronMail/releases/tag/v5.2.3

**Comment by i486** [comment]
> settings has check the bypass "... is not a win32 application"

You're most likely using old version since recent versions doesn't have that option.

> I have the same issue with latest version of ElectronMail 5.2.3 based on Electron 32.0.1
> 
> https://github.com/vladimiry/ElectronMail/releases/tag/v5.2.3

Works fine with VxKex 1.1.x

![image](https://github.com/user-attachments/assets/de75a71d-69e3-4f7e-8f1a-22d1f924272e)

**Comment by ask2018** [comment]
@i486 That is odd. It is not working for me. Still the same issue. Maybe you need to add account first?

**Comment by i486** [comment]
> @i486 That is odd. It is not working for me. Still the same issue. Maybe you need to add account first?

Can you provide reproducible steps?

**Comment by ask2018** [comment]
Well there is not much steps. I simply have the application with 4 Protonmail accounts. 5.2.2 works just fine with VxKex, but if I try 5.2.3 or 5.3.0 there is that error and I cannot run it at all. Here is screenshot. It is the same msg about not valid 32 application in Czech language, as the system is CZ.

![error-523](https://github.com/user-attachments/assets/0e64a180-cb71-4f67-92ea-ab686f1e3ae6)

**Comment by i486** [comment]
> Well there is not much steps. I simply have the application with 4 Protonmail accounts. 5.2.2 works just fine with VxKex, but if I try 5.2.3 or 5.3.0 there is that error and I cannot run it at all. Here is screenshot. It is the same msg about not valid 32 application in Czech language, as the system is CZ.
> 
> ![error-523](https://private-user-images.githubusercontent.com/35725232/398015638-0e64a180-cb71-4f67-92ea-ab686f1e3ae6.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ5MDY0NzQsIm5iZiI6MTczNDkwNjE3NCwicGF0aCI6Ii8zNTcyNTIzMi8zOTgwMTU2MzgtMGU2NGExODAtY2I3MS00ZjY3LTkyZWEtYWI2ODZmMWUzYWU2LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjIyVDIyMjI1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI1Zjk5YjgzNjQ1YzNmZDhlNGI2Yzc1YWU4NGFlMzg0ODNlOGRmOTBkNDAwY2M0Yzc2OGE5YTExM2ExMTU2ZjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.pAu8jH2GdiN1gtjy6lgU_Wo_LjA5dIGIHD1wR0dh3NM)

Which version of VxKex are you currently using? Could you try testing the latest version that was just released? It includes some fixes related to Electron.

Also what's your Windows 7 version? Language (Czech?) SP1? ESU Updates?

For me, both VxKex_Orig_1_1_1_1375 and VxKex_Orig_1_1_2_1428 work fine on 5.2.3 & 5.3.0.

**Comment by ask2018** [comment]
I've tried versions 1.1.1.1375, 1.1.2.1428 and as well unofficial version 1.1.2.1380 (that is the version I'm using normally, as none of the official versions support latest versions of Zoner Photo Studio). But for the test I've tried all the versions.
Windows version is Windows 7 x64 SP1 EN with Czech language pack. All ESU updates included. I have themes service fully disabled. It works for you as well if you add some account to the ElectronMail?
